[2025-04-13T22:35:52.597+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T22:35:52.603+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ihg_hotels_dag.load_to_snowflake manual__2025-04-13T22:32:04.434926+00:00 [queued]>
[2025-04-13T22:35:52.606+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ihg_hotels_dag.load_to_snowflake manual__2025-04-13T22:32:04.434926+00:00 [queued]>
[2025-04-13T22:35:52.606+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-04-13T22:35:52.610+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_to_snowflake> on 2025-04-13 22:32:04.434926+00:00
[2025-04-13T22:35:52.613+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=405) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-13T22:35:52.614+0000] {standard_task_runner.py:72} INFO - Started process 406 to run task
[2025-04-13T22:35:52.614+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ihg_hotels_dag', 'load_to_snowflake', 'manual__2025-04-13T22:32:04.434926+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/hotel_scrape_dag.py', '--cfg-path', '/tmp/tmprchwt1u2']
[2025-04-13T22:35:52.615+0000] {standard_task_runner.py:105} INFO - Job 209: Subtask load_to_snowflake
[2025-04-13T22:35:52.631+0000] {task_command.py:467} INFO - Running <TaskInstance: ihg_hotels_dag.load_to_snowflake manual__2025-04-13T22:32:04.434926+00:00 [running]> on host 599bf25ffed0
[2025-04-13T22:35:52.661+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ihg_hotels_dag' AIRFLOW_CTX_TASK_ID='load_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T22:32:04.434926+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-13T22:32:04.434926+00:00'
[2025-04-13T22:35:52.661+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-13T22:35:52.666+0000] {hotel_scrape_dag.py:287} INFO - Connecting to Snowflake...
[2025-04-13T22:35:52.667+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-13T22:35:52.667+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-13T22:35:52.667+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-13T22:35:54.873+0000] {hotel_scrape_dag.py:308} INFO - Creating stage and table in Snowflake...
[2025-04-13T22:35:55.351+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-13T22:35:55.622+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-13T22:35:55.623+0000] {hotel_scrape_dag.py:330} INFO - Copying data from S3 to Snowflake table...
[2025-04-13T22:35:56.867+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-13T22:35:57.027+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-04-13T22:35:57.029+0000] {hotel_scrape_dag.py:341} INFO - âœ… Data loaded into Snowflake table: hotels (0 rows)
[2025-04-13T22:35:57.029+0000] {connection.py:789} INFO - closed
[2025-04-13T22:35:57.076+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2025-04-13T22:35:57.135+0000] {python.py:240} INFO - Done. Returned value was: 0
[2025-04-13T22:35:57.148+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-13T22:35:57.148+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ihg_hotels_dag, task_id=load_to_snowflake, run_id=manual__2025-04-13T22:32:04.434926+00:00, execution_date=20250413T223204, start_date=20250413T223552, end_date=20250413T223557
[2025-04-13T22:35:57.165+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-13T22:35:57.173+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T22:35:57.173+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
